pkg/terraform/executor.go

```diff
@@ -211,22 +213,50 @@ func (ex *Executor) execute(verbose bool, args ...string) error {
 		wg.Add(1)
 
 		go tailFile(p, done, &wg)
+	} else {
+		fmt.Printf("\nYou can find the logs in %q\n", p)
```

Comment:
I'm not really sure if this is needed. Also, it uses `else` :smile:

Reply:
But then that is one of the requirement from the issue. What is wrong with using `else`?

Reply:
About using `else`: https://medium.com/@jamousgeorges/dont-use-else-66ee163deac3